src/Configuration/Config.php

```diff
@@ -115,27 +121,12 @@ public function getFileExtensions(): array
     {
         return $this->configuration['fileExtensions'] ?? self::FILE_EXTENSIONS_TO_PARSE;
     }
-
-    /**
-     * @param array $configuration The array containing the configuration values.
-     * @return void
-     */
-    private function validateConfigurationValues(array $configuration)
```

Comment:
Class was too long, inlining the method was the only way around the 200 lines limit.